### ISSUE TITLE
Can someone help me through discord?

### ISSUE THREAD

**Issue by Desuka-art** [PlayVoice/lora-svc#20]
I apologize, but some of the instructions are vague, and I need help with being able to do this. If someone can help me through discord or through here, that would be appreciated.

**Comment by MaxMax2016** [comment]
yes svc_preprocess_ids.py is missing,  you should use this https://github.com/PlayVoice/VI-Speaker;

**Comment by Desuka-art** [comment]
That's the problem. I have no idea where to put it, and I keep messing up and causing errors. Where do I put everything?

**Comment by MaxMax2016** [comment]
from VI-Speaker，you can get every folder's speaker embedding(must contain only one speaker's waves)，the speaker embedding is the average embedding of waves in the folder, and put speakers' embedding to data_svc as the date tree. 
data_svc/ids
|     └──spk1_encoding.npy
|     └──spk2_encoding.npy
|     └──spk3_encoding.npy

**Comment by MaxMax2016** [comment]
so sorry, i have no discord account.

**Comment by Desuka-art** [comment]
That's okay. Okay, let me try to ask this in more specific terms. What do I do once I download VI-Speaker? What python script do I use? Where do I put the waves? In the lora-svc place? Or in the VI-Speaker place? Also, about the speaker encoder, where do I put them? They appear to be six separate files, so I'm confused as to where I put them. I found out where to put the medium.pt, which would be just in the root of the lora-svc folder. So, I'm good there. I'm just confused on everything else.

**Comment by MaxMax2016** [comment]
What do I do once I download VI-Speaker? 
any where, go to root VI-Speaker ,and 'ln -s path_of_data_svc'
What python script do I use? 
python vi_speaker_batch.py ./saved_models/best_model.pth.tar ./saved_models/config.json ./data_svs/waves ./data_svc/ids
then you get the speaker embeddings.

**Comment by Desuka-art** [comment]
Yes, I've been to that link. Unfortunately, without knowing which version of numpy is supposed to be used, I have no idea what version to use.

**Comment by Desuka-art** [comment]
Okay, I figured out the issue, but it only gives me one file in the speaker_embedding_center folder. Is this right? I thought it would give me the individual files for the waves.

**Comment by Desuka-art** [comment]
So it turns out that, yes, it is supposed to give me only one thing. but now I have a new problem. When I use "python svc_preprocess_f0.py", it gives me the error "FileNotFoundError: [Errno 2] No such file or directory: './filelists/train.txt'" but nowhere in the directions does it say to make something like that. What do I do?

**Comment by MaxMax2016** [comment]
"mkdir filelists" first

**Comment by Desuka-art** [comment]
Understood. I see that train.txt is autofilled, but what about eval.txt? What do I put there?

**Comment by MaxMax2016** [comment]
cut the first 10 lines of train.txt to creat eval.txt

**Comment by Desuka-art** [comment]
I understand. And how long should my dataset be in hours/minutes? Just curious.

**Comment by Desuka-art** [comment]
TypeError: mel() takes 0 positional arguments but 2 positional arguments (and 3 keyword-only arguments) were given

I encountered this error suddenly when using VI-Speaker. I have no idea what to do.
